server/src/resolvers/reservationResolver.ts

```diff
@@ -0,0 +1,51 @@
+import { ReservationInput } from "../models/equipment/reservationInput";
+import { ReservationRepository } from "../repositories/Equipment/ReservationRepository";
+
+const reservationRepo = new ReservationRepository();
+
+const ReservationResolvers = {
+
+  Query: {
+    reservations: async (_: any, args: { Id: number }, context: any) => {
+      return await reservationRepo.getReservations();
+    },
+
+    reservation: async (_: any, args: { Id: number }, context: any) => {
+      return await reservationRepo.getReservationById(args.Id);
+    },
+
+    reservationByUser: async (_: any, args: { Id: number }, context: any) => {
+        return await reservationRepo.getReservationById(args.Id);
+    },
+    
+  },
+
+  Mutation: {
+
+    createReservation: async (_: any, args: { reservation: ReservationInput }) => {
+      return await reservationRepo.createReservation(args.reservation);
+    },
+
+    assignLabbieToReservation: async (_: any, args: { resId: number, labbieId: number }) => {
+      return await reservationRepo.assignLabbieToReservation(args.resId, args.labbieId);
+    },
+
+    removeLabbieFromReservation: async (_: any, args: { resId: number, labbieId: number }) => {
+      return await reservationRepo.removeLabbieFromReservation(args.resId);
```

Comment:
After discussion with Mike we decided to get rid of the concept of labbies being assigned to reservations, so I don't think we'll need `assignLabbieToReservation` or `removeLabbieFromReservation` anymore. Likewise we can remove the labbie stuff from the reservation table.